.gitignore

```diff
@@ -5,8 +5,7 @@ composer.phar
 composer.lock
 
 .phpunit.result.cache
+.phpunit.cache
 
 vendor/
 
-.idea/
```

Comment:
what is the reason behind this change?

Reply:
I have no .idea

Reply:
I think a code repository should not "depend" on any specific editor or IDE. Each developer has their own favourite editor that may leave artifacts in the folder. Ignoring those artifacts should be done in a [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)